recipe/meta.yaml

```diff
@@ -30,6 +30,7 @@ test: {}
 about:
   home: https://github.com/conda-forge/blas-feedstock
   license: BSD 3-clause
+  license_file: {{ environ.get('RECIPE_DIR', '') }}/LICENSE.txt
```

Comment:
Just as FYI, it doesn't work to use `get`. Probably because `environ` isn't defined in this context.